migrations/mysql/20250721140324_orders_add_uuid.sql

```diff
@@ -0,0 +1,9 @@
+-- +up
+-- +begin
+ALTER TABLE `orders` ADD COLUMN `uuid` VARCHAR(255) NOT NULL DEFAULT '';
```

Comment:
why using 255 chars here?

and it should be an optional field?

Reply:
If `uuid` is not given, it will be default to `''`.
And I just want to set a number that's long enough to accommodate a unique id string.

Reply:
fixed